index.html

```diff
@@ -59,11 +58,151 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                            <ul class="projectstack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </article>
+                   
+                
+                    <section class="myproject">
+                        <div class=" pheader">
+                            <img src="./img/mutitpostmain.png">
+                            
+                        </div>                        
+                        <div class="ptitle">
+                            <h3 >Multi-Post Stories </h3>
+                        </div>                        
+                        <uL class="details">                           
+                                
+                                <li class="conpoy"> CANOPY</li>                             
+                                <li class="backeddev">Back End Dev</li>                               
+                                <li class="text2015">2015</li> 
+                            </uL> 
+                        <h3>A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+                        <div class="lower_part">
+                            <ul class="projectstack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+            
+
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectthree.png">
+                            
+                        </div>
+                        <h3 class="ptitle">Tonic</h3>
+                        <uL class="details">                           
+                                
+                            <li class="conpoy"> CANOPY</li>                             
+                            <li class="backeddev">Back End Dev</li>                               
+                            <li class="text2015">2015</li> 
+                        </uL>  
+                        <h3>A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+                        <div class="lower_part">
+                            <ul class="project-tech-stack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
```

Comment:
- [ ] Kindly, try to match the Figma design as much as possible. Currently, the Work section is missing proper paddings/margins, styles, colors, etc.

Reply:
I have done copy of update in last few minutes

Reply:
Dear NatiG25 let me know if my update is effective. Thanks